config.js

```diff
@@ -1,7 +1,3 @@
-exports.DATABASE_URL = process.env.DATABASE_URL ||
-                       global.DATABASE_URL ||
-                      'mongodb://localhost/restaurants-app';
-exports.TEST_DATABASE_URL = (
```

Comment:
Why removing this?

Reply:
Removing `global.DATABASE_URL ||` since it is considered a bad practice and we don't actually use it anywhere. The end result will be...

`exports.DATABASE_URL = process.env.DATABASE_URL || 'mongodb://localhost/restaurants-app';`